.env.example

```diff
@@ -31,3 +31,6 @@ ACADEMIES_API_TIMEOUT=0.6
 #
 SENTRY_ENV=SENTRY_ENV # the environment production, pre-production/staging or development
 SENTRY_DSN=SENTRY_DSN # the DSN for the project in Sentry see: https://docs.sentry.io/platforms/ruby/configuration/options/#environment-variables
+
+# Redis
+REDIS_URL=REDIS_URL
```

Comment:
Could this be the default URL for running locally under Docker?

Reply:
I like the way the example has no values, the Redis URL could be any number of values so I am not sure I want to propose one at this level?